### DISCUSSION TITLE
Simple TIMER Query

### DISCUSSION THREAD

**Discussion by scargill** [arendst/Tasmota#13808] [Q&A] [answered]
I've not really used timers in Tsamota - I have a Tasmota'd Sonoff S26 (Version 10 development Tasmota)

I have pulsetime set to turn it off for 10 seconds then back on (inverted out) if set on... that works.

So by default the output is ON.

I just tried entering in the console at 11:48am on Friday (today):

Timer1 {"Enable":1,"Time":"11:00","Window":0,"Days":"SMTWTFS","Repeat":1,"Output":1,"Action":2}

At 11am that should surely have turned the output off for 10 seconds then back on - nothing happened. 

Ideas, could I be missing something - this is standard dev Tasmota..???

**Comment by sfromis** [comment, marked as answer]
To use timers, you should make sure to enable all timers, not only enable the individual timer. You can do that via the check box "Enable Timers" on the web UI page "Configure Timers", or via the command `Timers`.

**Reply by scargill** [reply]
I did thanks but could still not get the timer to do anything in the console - eventually I got timers running in the GUI (4 timers, 6 hours apart - and I'm using pulsetime - the point being to reset a less than perfect circuit every few hours automatically. Good enough but I must've missed something in my console attempts...

**Reply by sfromis** [reply]
One factor which may be confusing is that if you try to configure timers before having an output (like a relay) available, then you cannot configure a timer for on/off/toggle, which will be silently "upgraded" to the only thing able to work, rule. This also kicks in if you try to configure a timer using JSON and a command.
